avoid multiple NICs VM connectivity issue (#130)
harvester/docs: docs/networking/harvester-network.md

!!! note
    - Only the first NIC will be enabled by default. Users can either choose to use a management network or a VLAN network. 

>     - You will need to be careful to configure multiple NICs VM to avoid connectivity issue. You can refer to the [knowledge base](https://harvesterhci.io/kb/multiple-nics-vm-connectivity) for more details.

Reviewer: ```suggestion
    - You need to be careful to configure virtual machines with multiple NICs to avoid connectivity issues. You can refer to the [knowledge base](https://harvesterhci.io/kb/multiple-nics-vm-connectivity) for more details.
```

  - type: physical
    name: enp2s0
    subnets:
      - type: static
        address: 10.0.0.100/24 # IP address is varies upon your environment
```
Harvester is fully compatible with the `cloud-init network configs`. You can refer to the [documentation](https://cloudinit.readthedocs.io/en/latest/topics/network-config-format-v2.html) for more details.

!!! note

>     If adding additional NICs after VM started, you will need to configure IP for additional NICs manually.

Reviewer: ```suggestion
    If you add additional NICs after the VM has started, you will need to manually configure IPs for the additional NICs.
```
